executor/common_usb.h

```diff
@@ -289,23 +342,27 @@ static volatile long syz_usb_connect(volatile long a0, volatile long a1, volatil
 	debug("syz_usb_connect: device data:\n");
 	debug_dump_data(dev, dev_len);
 
-	struct usb_device_index index;
-	memset(&index, 0, sizeof(index));
+	int fd = usb_fuzzer_open();
+	if (fd < 0) {
+		debug("syz_usb_connect: usb_fuzzer_open failed with %d\n", fd);
+		return fd;
+	}
+	if (fd > MAX_FDS) {
+		debug("syz_usb_connect: too many open fds\n");
+		return -1;
+	}
+	debug("syz_usb_connect: usb_fuzzer_open success\n");
+
+	struct usb_device_index* index = &usb_indexes[fd];
```

Comment:
Looks like out-of-bounds write ;)
The check above needs to be >= MAX_FDS.

Reply:
Oops =)